src/lang/window.js

```diff
@@ -22,6 +22,19 @@
      */
     window.me = window.me || {};
 
+    /**
+    * Empty BaseClass constructor and prototype attached to 
+    * `me` global object to act as the base class for inheritance 
+    */
+    me.BaseClass = function () {};
+    me.BaseClass.prototype = Object.create(Object);
```

Comment:
`me.Object` is exactly what I had in mind, no changes there. If there's no technical reason that `me.Object` can't be the base class, I would prefer that.